UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Helpers/AuthTokenProvider.cs

```diff
@@ -30,26 +30,32 @@ public class AuthTokenProvider : IAuthEssTokenProvider, IAuthFssTokenProvider
 
         public async Task<string> GetManagedIdentityAuthAsync(string resource)
         {
-            var accessToken = GetAuthTokenFromCache(resource);
+            _logger.LogInformation(EventIds.GetAccessTokenStarted.ToEventId(), "Getting access token to call external endpoint started | {DateTime} | _X-Correlation-ID:{CorrelationId}", DateTime.Now.ToUniversalTime(), CommonHelper.CorrelationID);
 
+            AccessTokenItem? accessToken = GetAuthTokenFromCache(resource);
             if (accessToken != null && accessToken.AccessToken != null && accessToken.ExpiresIn > DateTime.UtcNow)
```

Comment:
We are using SlidingExpiration to configure the token's expiration to occur one minute before the real expiration time.
This resolves the situation because it is likely that the new access token will be issued before the expiration time.

SlidingExpiration:- will expire the entry if it hasn't been accessed in a set amount of time.
AbsoluteExpiration:- will expire the entry after a set amount of time.